modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/index.es.js

```diff
@@ -12,21 +12,17 @@
  * details.
  */
 
-import process from 'process';
+(function() {
+	const getModule = window[Symbol.for('__LIFERAY_WEBPACK_GET_MODULE__')];
 
-export {default as render} from './render.es';
-export {default as useEventListener} from './hooks/useEventListener.es';
-export {default as useInterval} from './hooks/useInterval.es';
-export {default as useIsMounted} from './hooks/useIsMounted.es';
-export {default as usePrevious} from './hooks/usePrevious.es';
-export {default as useStateSafe} from './hooks/useStateSafe.es';
-export {default as useThunk} from './hooks/useThunk.es';
-export {default as useTimeout} from './hooks/useTimeout.es';
-
-// Egregious hack because react-dnd expects `window.process` to exist:
-//
-// https://github.com/react-dnd/asap/blob/b6bebeb734/src/node/asap.ts#L24
-
-if (!window.process) {
-	window.process = process;
-}
+	getModule('frontend-js-react-web').then(
```

Comment:
Can this promise reject? If so, we should have a catch so as to avoid unhandled promise rejection spam in the console.

Reply:
Yes, it can be rejected. :+1: